yax/core/src/main/scala/doobie/util/compat.scala

```diff
@@ -63,7 +63,14 @@ object compat {
 
 #+fs2
     object fs2 {
-      import _root_.fs2.util.{ ~> => Fs2Nat }
+      import _root_.fs2.util.{ Monad => Fs2Monad, ~> => Fs2Nat }
+
+      implicit def monadCompat[F[_]](implicit monad: Monad[F]): Fs2Monad[F] =
```

Comment:
Yeah this is related to #311 as well. Let's leave it for now and fix compat in another PR.